File: Array va String /SonlarniBirlashtirish.py
"""
Ikki arrayda tartiblangan sonlar keltirilgan.
Ularni tartiblangan ko'rinishda birlashtiring.
Misol uchun:
  merge_two_arrays([4,5], [3]) -> [3,4,5]
  merge_two_arrays([1,3], [2,6]) -> [1,2,3,6]
"""

def merge_two_arrays(arr1, arr2) -> array:
    """
    Kodni bu yerda yozing.
    """
    i = j = 0
    n, m = len(arr1), len(arr2)
    result = []
    
    while i < n and j < m:
        if arr1[i] < arr2[j]:
            result.append(arr1[i])
            i += 1
        else:
            result.append(arr2[j])
            j += 1
            
    while i < n:
        result.append(arr1[i])
        i += 1
        
    while j < m:
        result.append(arr2[j])
        j += 1
    
    return result

print(merge_two_arrays([1,2,3,3],[5,8,9]))

File: Array va String /SonlarniBirlashtirish.cs
int[] merge_two_arrays(int[] arr1, int[] arr2)
{
    var i = 0;
    var j = 0;
    var n = arr1.Length;
    var m = arr2.Length;
    var result = new int[n + m];
    
    while(i < n && j < m)
    {
        if(arr1[i] < arr2[j])
        {
            result.Append(arr1[i]);
            i++;
        }
        else
        {
            result.Append(arr2[j]);
            j++;
        }     
    }
    while(i < n)
    {
        result.Append(arr1[i]);
        i++;
    }
    while(j < m)
    {
        result.Append(arr2[j]);
        j++;
    }

    return result;
    
}

var res = merge_two_arrays(new int[] {1,3,5,7,8,11}, new int[] {2,4,6,9,10});
Array.ForEach(res, Console.WriteLine);